.identity/03_github_environment.tf

```diff
@@ -120,3 +120,12 @@ resource "github_actions_environment_secret" "secret_integration_test_ehub_tx_ne
   plaintext_value  = data.azurerm_key_vault_secret.key_vault_integration_ehub_tx_negative_biz_key[0].value
 }
 
+#tfsec:ignore:github-actions-no-plain-text-action-secrets # not real secret
```

Comment:
Maybe this is a typo: it is a real secret.

Reply:
Copied from previous secrets, deleting it